Speech/samples/V1p1beta1/SpeechQuickstartBeta.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * DO NOT EDIT! This is a generated sample ("Request",  "speech_quickstart_beta")
+ */
+
+// sample-metadata
+//   title:
+//   description: Performs synchronous speech recognition on an audio file.
+//   usage: php samples/V1p1beta1/SpeechQuickstartBeta.php [--sample_rate_hertz 44100] [--language_code "en-US"] [--uri_path "gs://cloud-samples-data/speech/brooklyn_bridge.mp3"]
```

Comment:
/cc @bcoe

Would it be possible to split the parameters on to new lines?

Reply:
I can check if it's possible to do this in the generator.

Reply:
I think I need to understand more about how the metadata is used to have a better opinion about whether/how to fix this in the generator @bcoe @beccasaurus .

- Would the tool that parses the metadata be able to parse multi-line `usage`?

- Would the tool that parses the metadata needs the optional flags? (AFAIK they are mostly for testing?) If not, we can just drop them and the line will very likely fit in character limits.

Fighting the generator for formatting is fundamentally hard (but doable). If we don't need the sample metadata right away or for what they are used for formatting is tolerable, I suggest we not block on the formatting. After all, they are outside the region tag and are not displayed on devsite.